background/constants/networks.ts

```
@@ -724,5 +700,6 @@ export function ShardFromRpcUrl(url: string): string {
       }
     }
   }
-  throw new Error("Unknown shard for rpc url: " + url)
+  logger.error("Unknown shard for rpc url: " + url)
+  return ""
```

hmm, shouldn't this still throw an error? The caller doesn't check the failure case

---

Because currently RPCs from other chains are sent into this function, throwing an error will break logic that would otherwise perform 'acceptably'. When we remove RPCs from other chains, I agree, it should throw an error.

---

> When we remove RPCs from other chains, I agree, it should throw an error

Ok, please create an issue and link back to this thread

---

https://github.com/PelagusWallet/pelagus/issues/5